pine/src/main/java/com/distkv/pine/components/PineHandle.java

```diff
@@ -0,0 +1,25 @@
+package com.distkv.pine.components;
+
+import com.distkv.common.id.PineHandleId;
+
+/**
+ * The abstract class `PineHandle` is used to represent
+ * a handle of the Pine component.
+ */
+public abstract class PineHandle {
```

Comment:
it's better to make class renamed to `AbstractPineHandle`.

Reply:
Good point.